src/stores/Widgets/WidgetsStore.ts

```diff
@@ -961,6 +994,16 @@ export class WidgetsStore {
             previousPageButton.attr("style", config.currentImagePage > 0 ? "" : "cursor: not-allowed; opacity: 0.2");
             previousPageButton.attr("title", config.imagePanelMode === ImagePanelMode.None ? "previous image" : "previous page");
         }
+
+        const imagePanelButton = $(".lm_goldenlayout")?.find("li.lm-image-panel[style!='display:none;']");
+        if (imagePanelButton) {
+            imagePanelButton.attr("style", channelMapStore.channelMapEnabled ? "opacity: 0.2" : "");
```

Comment:
I suggest keeping the image panel button and the channel map button responsive (showing a darker color when hovered) when they are clickable.